### ISSUE TITLE
Generate v20.1.4 files

### ISSUE THREAD

**Issue by ericmschmidt** [augustoproiete/DevExpress-NuGet#26]
Would you to generate the scripts to build the v20.1.4 packages? Or have you published the code you use to create these and I can modify and generate them?

**Comment by augustoproiete** [comment]
@ericmschmidt I haven't been using DevExpress controls in a while, and I had the impression that for v20 they were going to add support for .NET Core 3 and also release NuGet packages for all their controls. Isn't that the case?

**Comment by ericmschmidt** [comment]
It does look like they started supporting .NET Core 3.0 starting in v19.1 and started supporting .NET Core 3.1 (LTS) starting in v19.2.5, however it does still look like they support .NET Framework 4.5.2 as well.

As for the NuGet packages yes they do have a NuGet feed but it appears they've grouped DLLs into packages, and don't have the 1-1 relationship that your scripts generate. I'm sure there are some scenarios where this makes sense but there are definitely some where it doesn't. But if you've moved away from the controls I'm guessing you don't really want to support generating the scripts moving forward either, is that accurate?

**Comment by augustoproiete** [comment]
Thanks for confirming @ericmschmidt.

If there's value in having the 1:1 package relationship still, I'm happy to re-generate the nuget specs. I'll book some time next week to re-generate the nuget specs at least for the most recent couple versions of v19.* and v20.* and book some time in July to open-source a dotnet tool to generate the NuGet specs

**Comment by ericmschmidt** [comment]
Yes that would still be handy for us our team at this point. Thank you for your continued support on this.

**Comment by jansohn** [comment]
@augustoproiete any update on this?
